src/main/java/net/minecraftforge/client/event/ModelEvent.java

```diff
@@ -34,7 +34,53 @@ protected ModelEvent()
 
     /**
      * Fired when the {@link ModelManager} is notified of the resource manager reloading.
-     * Called after model registry is set up, but before it's passed to {@link net.minecraft.client.renderer.block.BlockModelShaper}.
+     * Called after the model registry is set up, but before it's passed to the
+     * {@link net.minecraft.client.renderer.block.BlockModelShaper}.
```

Comment:
Please modify this javadoc so it's first sentence clearly differentiates its purpose from `ModelEvent.BakingCompleted` (such as combining both sentences into one, or simply discarding the first sentence and using the second one with a bit of rewriting).